DevEnvSetup.md

```diff
@@ -0,0 +1,31 @@
+Mod Manager Development Environment Setup Information
+Don't trust versions in any .md file or other READMEs. Even this one. Those might not be correct.

```

Comment:
I'd take this line out as you mention versions later. It doesn't provide any solution to the problem so it's just kind of a useless line as if you get a version issue then you're probably stuck anyway